ansible/roles/sunbird-system-init/templates/system_init_commands.j2

```diff
@@ -0,0 +1,344 @@
+#!/bin/sh
+
+validateConfigFields() {
+        if [ -z "{{sunbird_custodian_tenant_name}}" ] || [ -z "{{sunbird_custodian_tenant_description}}" ] || \
```

Comment:
Can the Ansible variables (enclosed in double curly braces) be defined as normal shell variables so that it is easy to validate the system init script on a laptop?

Reply:
Review Comment Incorporated in code